### PR TITLE
Various cleanup items

### PR DESCRIPTION
**Pull request by csandman**
While I was looking through the code for this project again, I found a few cleanup items that would probably be good to get into the project!

1. In a recent commit, you accidentally renamed the actual form field `formFieldPaperWidth` to the string `"formFieldPaperWidth"`. I fixed it by changing it back.
2. After looking through the docs for a while, I couldn't find any mention of `googleChromeRpccBufferSize`. I'm fairly certain that form option was removed, and setting it no longer does anything.
3. The field `pageRanges` doesn't exist anymore, and has been renamed to `nativePageRanges`. I updated everything in the code to use that, except I left the function name `PageRanges` the same so it's not technically a breaking change.
4. In the test files, I was getting a few lint errors for unhandled errors, so I just added the line `require.Nil(t, err)` where it was missing:
image

And besides those, there were a few other changes that I think just improve the quality of the project in general.

First, I updated all of the form field function comments to actually reflect what the [page properties documentation](https://gotenberg.dev/docs/routes#page-properties-chromium) describe them as. A description like "Landscape sets landscape form field." is pretty useless to an end user, so describing what the end result of each function is provides a much better experience IMO.

Second, I added a few additional default `PaperDimensions` objects. In the docs they have this section providing a list of default paper dimensions, so I felt like it made sense to at least offer all of those:

image

And finally, I changed the argument of the `Format()` function into a named string type, `ImageFormat`. This field can only be set to one of the three constant values, so it makes sense to restrict it to one of those. Also, I realized this function was actually defined incorrectly anyway, so this change fixes that:

```go
// Format sets format form field
func (req *chromeRequest) Format(format string) {
	req.values[formFieldFormat] = format
	if format == "" {
		req.values[format] = JPEG // <- this line is saying, if the passed `format` is an empty string, set req.values[""] = JPEG
	}
}
```

By restricting the argument to the typed string, you don't even have to deal with a default, because the user can no longer pass an empty string as an argument.

---

Let me know if all of these changes sound good to you! If not, feel free to cherry pick the ones you actually want. I feel like at least the fixes should definitely be merged though haha.

### CODE REVIEW

File: chrome.go [hunk 1]
```diff
@@ -7,23 +7,22 @@ import (
 )
 
 const (
-	formFieldWaitDelay                  string = "waitDelay"
-	formFieldPaperWidth                 string = "formFieldPaperWidth"
```
Comment (csandman) [inline chrome.go:11]: It looks like this was renamed in a bad find/replace

File: chrome.go [hunk 2]
```diff
@@ -181,59 +205,54 @@ func (req *chromeRequest) Margins(margins PageMargins) {
...
+// Default is 1.0.
 func (req *chromeRequest) Scale(scaleFactor float64) {
 	req.values[formFieldScale] = fmt.Sprintf("%f", scaleFactor)
 }
 
-// SkipNetworkIdleEvent sets skipNetworkIdleEvent form field as true.
+// Do not wait for Chromium network to be idle.
 func (req *chromeRequest) SkipNetworkIdleEvent() {
-	req.values[formFieldSkipNetworkIdleEvent] = "true"
+	req.values[formFieldSkipNetworkIdleEvent] = strconv.FormatBool(true)
```
Comment (csandman) [inline chrome.go:226]: Oh, the one other change was just to change these from static strings to `strconv.FormatBool(true)`. This is kind of pointless, but it seems like a more strict approach to define them like this, as there is no room for mistyping the word "true" haha.

File: chrome.go (hunk 1 above)
Comment (dcaraxes) [reply chrome.go:11]: Hi! Yes, indeed, this happened after the renaming to GoLand and requires correction.

File: chrome.go (hunk 2 above)
Comment (dcaraxes) [reply chrome.go:226]: I agree, this looks much better

**Pull request merged**